api/public/v2/tests/test_flag_viewset.py

```diff
@@ -38,15 +66,64 @@ def _request_flags(self):
         )
         return self.client.get(url)
 
-    def test_flag_list(self, get_repo_permissions):
+    def test_flag_list_no_commit(self, get_repo_permissions):
+        get_repo_permissions.return_value = (True, True)
+
+        res = self._request_flags()
+        assert res.status_code == 200
+        assert res.json() == {
+            "count": 2,
+            "next": None,
+            "previous": None,
+            "results": [
+                {"flag_name": "foo", "coverage": 0},
```

Comment:
Jw on returning 0 for coverage here vs. None 🤔

Reply:
yeah I feel none is better